package.json

```diff
@@ -44,8 +44,9 @@
     "chalk": "^1.1.3",
     "del": "^2.2.0",
     "execa": "^0.4.0",
+    "has-yarn": "^1.0.0",
     "inquirer": "^1.2.1",
-    "listr": "^0.6.1",
+    "listr": "^0.10.0",
```

Comment:
I agree, ping @gillchristian - or better yet, could you just give me write access to that repo?